services/provider/proto/provider.proto

```diff
@@ -169,6 +171,9 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
+    // DEPRECATED: Contains subscription channel of provider operator for client operator to match
```

Comment:
`[deprecated = true];` need to be added in front of the field to mark is deprecated.

Reply:
Done as per the doc https://protobuf.dev/programming-guides/proto3/ so resolving.